.github/workflows/ci.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
       matrix:
         os:
           - ubuntu-20.04
+          - macos-latest
```

Comment:
Also if were going to go for this, we should use a specific macos release, otherwise the builds will not be reproducible in the future (we already had issues trying to do patch releases for old versions and the CI failing because its pulling newer dependencies that don't work with the old code).